tf_encrypted/convert/register.py

```diff
@@ -138,11 +146,13 @@ def _conv2d(converter, node, inputs):
 
   fmt = node.attr["data_format"].s.decode('ascii')
 
-  layer = Conv2D(x_in.shape.as_list(),
-                 shape,
-                 strides=int(max(node.attr["strides"].list.i)),
-                 padding=node.attr["padding"].s.decode('ascii'),
-                 channels_first=fmt == "NCHW")
+  layer = Conv2D(
+      x_in.shape.as_list(),
+      shape,
+      strides=int(max(node.attr["strides"].list.i)),
+      padding=node.attr["padding"].s.decode('ascii'),
+      channels_first=fmt == "NCHW"
```

Comment:
any reason why yapf isn't formatting these last lines in a standard way? sometimes it adds commas (which I think is preferable), but some like this one are missing one

Reply:
yapf doesn't add commas on its own, you have to manually do that, which in turn forces this formatting. however, it may choose to go with this formatting either way if the line will otherwise become too long. the Google style is less likely to do this but the Facebook style does it by default; we are somewhere in-between thanks in part to `dedent_closing_brackets=True`).

with black on the other hand, there doesn't seem to be a way to force it to use this formatting, and it may decide to keep everything on a single line after one line break, which is not ideal either imho.

Reply:
yeah, that seems strange given black's stated goal of reducing diff churn. overall I'm okay with both if you are!